src/pages/en/guides/deploy/vercel.mdx

```diff
@@ -76,3 +76,10 @@ After your project has been imported and deployed, all subsequent pushes to bran
 3. When asked `Want to override the settings? [y/N]`, choose `N`.
 4. Your application is deployed! (e.g. [astro.vercel.app](https://astro.vercel.app/))
 
+
+### Project config with vercel.json
+
+The `vercel.json` configuration file lets you configure, and override the default behavior of Vercel from within your project.
+Additionaly it is also possible to attach headers to HTTP responses from your Deployments. 
+
+📚 Learn more about Vercel’s [Project config](https://vercel.com/docs/project-configuration)
```

Comment:
```suggestion
📚 Learn more about [Vercel’s project configuration](https://vercel.com/docs/project-configuration).
```
Just want to make sure the link description is clear that this is for Vercel's project config!